DESCRIPTION

```diff
@@ -40,8 +40,8 @@ Suggests:
     rmarkdown,
     rpart,
     testthat
-Remotes: 
-    tidymodels/parsnip
+Remotes:
+    tidymodels/parsnip#631
```

Comment:
do we merge the parsnip PR first so that we can use the main branch of parsnip as the remote?

Reply:
The other PRs has been closed https://github.com/tidymodels/plsmod/pull/21, but you are free to wait for the PR to be merged before merging this one.